pypeit/coadd2d.py

```diff
@@ -605,7 +675,7 @@ def create_pseudo_image(self, coadd_list):
             if None not in all_maskdef_ids:
                 maskdef_id[islit] = coadd_dict['maskdef_id']
                 maskdef_objpos[islit] = coadd_dict['maskdef_objpos']
-                maskdef_slitcen[:, islit] = np.full(nspec_pseudo, coadd_dict['maskdef_slitcen'])
+                maskdef_slitcen[:, islit] = np.full(nspec_pseudo, coadd_dict['maskdef_slitcen'] + slit_left[:,islit])
```

Comment:
Has this maskdef_slitcen been redefined to be an offset from the left slit boundary?